web/src/main/webResources/WEB-INF/config-db/jndiExamples_tomcat/oracle_JNDI_Example.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (C) 2001-2016 Food and Agriculture Organization of the
+  ~ United Nations (FAO-UN), United Nations World Food Programme (WFP)
+  ~ and United Nations Environment Programme (UNEP)
+  ~
+  ~ This program is free software; you can redistribute it and/or modify
+  ~ it under the terms of the GNU General Public License as published by
+  ~ the Free Software Foundation; either version 2 of the License, or (at
+  ~ your option) any later version.
+  ~
+  ~ This program is distributed in the hope that it will be useful, but
+  ~ WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  ~ General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU General Public License
+  ~ along with this program; if not, write to the Free Software
+  ~ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
+  ~
+  ~ Contact: Jeroen Ticheler - FAO - Viale delle Terme di Caracalla 2,
+  ~ Rome - Italy. email: geonetwork@osgeo.org
+  -->
```

Comment:
can we provide some guidance here, like:
- move this file into tomcat/conf folder (I guess this is the step required?)
- fill relevant connection parameters
- read more about jndi in tomcat at ...

Reply:
Sounds good, will update the PR. Part of this is already in the file  `README.md`, we can improve the file and referenced it from these samples if sounds good.

Reply:
Sounds good, part of these instructions are in `README.md` file, we can improve it and reference it from the sample files

Reply:
Added additional documentation and example: https://github.com/geonetwork/core-geonetwork/pull/4599/commits/f10153223a5d56e0dc5bbb85ccc755587a1c4269

Reply:
thanx!